CHANGELOG

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
+https://docs.docker.com/engine/deprecated/ where target removal dates can also
+be found.
+
+## 17.10.0-ce (2017-10-DD)
+
+### Builder
+
+* Reset uid/gid to 0 in build context to fix cache busting issues on `ADD/COPY` [docker/cli#513](https://github.com/docker/cli/pull/513)
+- Fix build with `ADD` urls without any sub path [moby/moby#34217](https://github.com/moby/moby/pull/34217)
+- Fix loader error with different build syntax [docker/cli#544](https://github.com/docker/cli/pull/544)
+
+### Client
+
+* Move output of `docker stack rm` to stdout [docker/cli#491](https://github.com/docker/cli/pull/491)
+* Remove deprecated `--enable-api-cors` flag [moby/moby#34821](https://github.com/moby/moby/pull/34821)
+* Use nature sort secrets and configs in cli [docker/cli#307](https://github.com/docker/cli/pull/307)
+* Use non-detached mode as default for `docker service` commands [docker/cli#525](https://github.com/docker/cli/pull/525)
+* Set APIVersion on the client, even when Ping fails [docker/cli#546](https://github.com/docker/cli/pull/546)
+
+### Logging
+
+* Move pkg/jsonlog to be a subpackage of the single consumer [moby/moby#34946](https://github.com/moby/moby/pull/34946)
+- Fixed `raw` mode splunk logger [moby/moby#34520](https://github.com/moby/moby/pull/34520)
+
+### Networking
+
+* Ensure Host Network Service exists [moby/moby#34928](https://github.com/moby/moby/pull/34928)
+* Enabling ILB/ELB on windows using per-node, per-network LB endpoint. [moby/moby#34674](https://github.com/moby/moby/pull/34674)
+
+### Runtime
+
+* LCOW: Add UVM debugability by grabbing logs before tear-down [moby/moby#34846](https://github.com/moby/moby/pull/34846)
+* LCOW: Fix comment in graphdriver code [moby/moby#34891](https://github.com/moby/moby/pull/34891)
+* LCOW: Prepare work for bind mounts [moby/moby#34258](https://github.com/moby/moby/pull/34258)
+* LCOW: Support for docker cp, ADD/COPY on build [moby/moby#34252](https://github.com/moby/moby/pull/34252)
+* LCOW: VHDX boot to readonly. [moby/moby#34754](https://github.com/moby/moby/pull/34754)
```

Comment:
Perhaps we should combine all the LCOW lines to something like: "Add initial support for Linux Containers On Windows (LCOW)" - or better wording

@friism ^^